modules/v1/users/resources/UserResource.php

```diff
@@ -42,14 +54,116 @@ public function fields()
 
     public function extraFields()
     {
-        return ['userProfile'];
+        return ['userDepartments'];
     }
 
     /**
      * @return ActiveQuery
      */
-    public function getUserProfile()
+    public function getUserDepartments()
+    {
+        return $this->hasMany(UserDepartmentResource::class, ['user_id' => 'id']);
+    }
+
+    /**
+     * @return string[][]
+     */
+    public static function getUserRoles() {
+        return [
+            [
+                'value' => 'user',
```

Comment:
This should not be like this.
We should have value to be constant and text must be user text written in Yii:t( method.